### ISSUE TITLE
[BUG] ignore <noscript> when truncated feed content

### ISSUE THREAD

**Issue by math-GH** [FreshRSS/FreshRSS#3688]
**Describe the bug**

**To Reproduce**
Feed URL:
https://www.spiegel.de/index.rss

CSS selector `#Inhalt section.relative`
![grafik](https://user-images.githubusercontent.com/1645099/124666479-67ad0480-deae-11eb-8ada-fab0b6b3d577.png)

It comes to doubled pictures
![grafik](https://user-images.githubusercontent.com/1645099/124981503-f0a27800-e035-11eb-9d06-a195bdb69a67.png)
![grafik](https://user-images.githubusercontent.com/1645099/124981588-06b03880-e036-11eb-8b32-b61c96321d05.png)
(URL of article: https://www.spiegel.de/ausland/haiti-jovenel-moise-ermlordet-staat-der-gewalt-a-2b30564e-ff09-44ae-a4e4-94f1ee78ef44#ref=rss )

The reason:
The image is also in `<noscript>`
![grafik](https://user-images.githubusercontent.com/1645099/124981799-4414c600-e036-11eb-89b3-df9c5a9799be.png)

Conclusion:
The function, that extracts the content deletes the `<nosript>` and so the content of `<noscript>` is shown

**Expected behavior**
ignore `<noscript>` content or have an option in to ignore or show it

**Environment information (please complete the following information):**
 - FreshRSS version: [e.g. 1.18.1]

**Comment by Alkarex** [comment]
Could you try to remove `<noscript>` from this list?

https://github.com/FreshRSS/FreshRSS/blob/dff31cf55712758dd49ba333399dbc3478321f73/lib/lib_rss.php#L202

**Comment by math-GH** [comment]
it works.

should I open a PR?

**Comment by Alkarex** [comment]
@mathContao Yes, you are welcome. We need to evaluate potential usability or security issues. Especially if we strip `<script>` but not `<noscript>`

**Comment by Alkarex** [comment]
Looking more closely at that feed, here is a snippet:

```html
<span class="absolute top-0 left-0 w-full max-h-screen">
	<noscript>
		<img class="block h-full mx-auto" src="https://cdn.prod.www.spiegel.de/images/7f755417-0001-0004-0000-000001441033_w568_r0.9114893617021277_fpx55_fpy44.6.jpg" srcset="https://cdn.prod.www.spiegel.de/images/7f755417-0001-0004-0000-000001441033_w284_r0.9114893617021277_fpx55_fpy44.6.jpg 284w, https://cdn.prod.www.spiegel.de/images/7f755417-0001-0004-0000-000001441033_w335_r0.9114893617021277_fpx55_fpy44.6.jpg 335w, https://cdn.prod.www.spiegel.de/images/7f755417-0001-0004-0000-000001441033_w568_r0.9114893617021277_fpx55_fpy44.6.jpg 568w" width="568" height="623" sizes="568px" loading="lazy" alt>
	</noscript>
	<img data-image-el="img" class="block lazyload h-full mx-auto" src="https://cdn.prod.www.spiegel.de/images/7f755417-0001-0004-0000-000001441033_w568_r0.9114893617021277_fpx55_fpy44.6.jpg" width="568" height="623" sizes="568px" srcset="data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw==" loading="lazy" data-srcset="https://cdn.prod.www.spiegel.de/images/7f755417-0001-0004-0000-000001441033_w284_r0.9114893617021277_fpx55_fpy44.6.jpg 284w, https://cdn.prod.www.spiegel.de/images/7f755417-0001-0004-0000-000001441033_w335_r0.9114893617021277_fpx55_fpy44.6.jpg 335w, https://cdn.prod.www.spiegel.de/images/7f755417-0001-0004-0000-000001441033_w568_r0.9114893617021277_fpx55_fpy44.6.jpg 568w" alt>
</span>
```

I believe the feed is wrong because the second image is always displayed and does not depend on supporting `<script>`, so the feed should not have that `<noscript>` section at all.

As far as I can see, the current behaviour of FreshRSS is correct.

But to solve your issue more generally, I think the proper fix would be to implement a native filter, allowing to remove elements matching a CSS Path / XPath.

**Comment by Alkarex** [comment]
While waiting for the proper fix, the easiest is to make a mini FreshRSS extension for the problematic feed(s)
